src/ts/playerutils.ts

```diff
@@ -60,26 +60,25 @@ export namespace PlayerUtils {
   }
 
   /**
-   * Returns seekable time range.
-   * For live streams it will calculate timestamp timerange if tweak
-   * `enable_seek_for_live` is not present
+   * Calculates player seekable time range for live.
+   * For live streams we need to calculate range since we receive { start: -1, end: -1 }
+   * unless `enable_seek_for_live` tweak is present
+   *
+   * @param player
    */
-  export function getSeekableRange(player: PlayerAPI): TimeRange {
-    let {start, end} = player.getSeekableRange();
-
-    // If it is VOD or we have `enable_seek_for_vod` enabled
-    if (!player.isLive() || end > -1 && start > -1) {
-      return {start, end};
+  export function getSeekableRangeForLive(player: PlayerAPI): TimeRange {
```

Comment:
naming suggestion: `getSeekableRangeRespectingLive` as it covers `VOD` and `Live` case. Still, implicit knowledge needed 🤔

Reply:
Done in https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/b288dd3592345f25efc4f4da516e4e526a79da49